src/sass/components/_header.scss

```diff
@@ -126,6 +126,108 @@ header {
       height: 84px;
     }
 
```

Comment:
Don't mixed media-queries/RWD with plain scss.
You have got _media-queries file in src/sass directory, please move your code there.
Create header selector in intrested range and put it there.

Reply:
resolved